.sidekick.toml

```diff
@@ -30,7 +30,7 @@ protobuf-subdir             = "src"
 [codec]
 # The default version for all crates. This can be overridden in the crate's
 # `.sidekick.toml` file.
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
The current release is 0.1.0. We already bumped the version number.  I do not think we need to bump it again.

There are also references to this version number starting on line 60.  If we decide to update the number here, we have to update those numbers too.

Reply:
D'oh, I messed up my commits. This change was reverted in the "generated" commit.

there are no changes to `.sidekick.toml`.

Reply:
PTAL.